wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -66,9 +71,16 @@ def _check_base_classes_number(self, node: ast.ClassDef) -> None:
         """Check number of base classes."""
         if len(node.bases) > self.options.max_base_classes:
             self.add_violation(
-                TooManyBaseClassesViolation(node, ext=node.name),
+                TooManyBaseClassesViolation(node, text=node.name),
             )
 
+    def _check_yield_inside_init(self, node: AnyFunctionDef) -> None:
+        """Check yield is not in the __init__."""
+        if node.name == '__init__':
```

Comment:
We have a constant for `__init__`: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/constants.py#L161

Please, use it.